owm2ffmap/owm2ffmap.py

```diff
@@ -223,6 +249,7 @@ def process_node_json(comment, body, hostid=None, firstseen=None, lastseen=None)
                     #'network': {'addresses': False, #TODO
```

Comment:
@sarumpaet  the "network"-array here seems just commented out. Did you take it from the example but decided to not fill for our stats?
Probably we can use it for the mesh-link below.